presto-native-execution/presto_cpp/main/http/HttpClient.cpp

```diff
@@ -472,6 +478,9 @@ folly::SemiFuture<proxygen::HTTPTransaction*> HttpClient::createTransaction(
         nullptr,
         nullptr);
     sessionPool_->putSession(session);
+    RECORD_HISTOGRAM_METRIC_VALUE(
+        velox::kMetricExchangeTransactionCreateDelay,
```

Comment:
It is weird that counter `kMetricExchangeTransactionCreateDelay` is declared in Velox, but fulfilled in Prestissimo code.
Seems wrong.